Controller/CoinsController.cs

```diff
@@ -35,18 +33,21 @@ public void EarnCoins(object sender, EventArgs e)
             if(counter == 1000)
             {
                 counter = 0;
-                AddCoins(UserController.CurrentUser);
+                AddCoins();
             } 
         }
 
-        public void AddCoins(User user, int coins = 1)
+        public void AddCoins(int coins = 1)
         {
+            var userController = UserController.Create(new DatabaseContext());
+            var user = userController.GetItem(UserController.CurrentUser.ID);
             user.Coins += coins;
             userController.UpdateItem(user);
         }
 
         public void RemoveCoins(User user, int coins = 1)
         {
+            var userController = UserController.Create(new DatabaseContext());
```

Comment:
Wordt deze controller niet door een test getest?
Want dan mag je hier geen `new DatabaseContext()` gebruiken.

Reply:
Dit is alleen gemaakt zodat we geen foutmelding zouden krijgen, Ik weet niet precies wie hiermee bezig moest gaan maar dat moest later gefixed worden @VincentVinTin maybe?

Reply:
Maar goed niet onze taak!